Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md

```diff
@@ -178,14 +181,18 @@ A message created for this error.
   graph._drop("myGraph", true);
 @END_EXAMPLE_ARANGOSH_RUN
 
-@EXAMPLE_ARANGOSH_RUN{HttpGharialCreate2}
+Create a SmartGraph. This graph uses 9 shards and
+is sharded by the "region" attribute.
```

Comment:
Should it be mentioned here that the example will only work in the Enterprise Edition?

Reply:
yes, please add "Available in the Enterprise Edition only."